### ISSUE TITLE
Enable mobile next button in registration form

### ISSUE THREAD

**Issue by atd** [P2Pvalue/teem#261]
We should support using the mobile next button in the registration form

**Comment by atfornes** [comment]
did you mean back button, @atd?

Is the bug still relevant?

**Comment by atd** [comment]
No, I mean the next button that appears in some keyboard versions

I saw a user trying to use it in the registration form
